### PR TITLE
OMID-277 Omid 1.1.2 fails with Phoenix 5.2

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -293,6 +294,67 @@
...
+                <plugin>
+                  <groupId>org.apache.maven.plugins</groupId>
+                  <artifactId>maven-enforcer-plugin</artifactId>
+                  <dependencies>
+                    <dependency>
+                      <groupId>de.skuzzle.enforcer</groupId>
+                      <artifactId>restrict-imports-enforcer-rule</artifactId>
+                      <version>${restrict-imports.enforcer.version}</version>
+                    </dependency>
+                  </dependencies>
```
Comment (virajjasani) [inline pom.xml:306]: Nice! When i started looking at the changes, i immediately came here to see if we are going to restrict the import.

**Review by virajjasani — COMMENTED** [review]
Here we need manual verification only with locally deployed cluster?

File: pom.xml (hunk 1 above)
Comment (stoty) [reply pom.xml:306]: Straight copy-paste from Phoenix :)

**Comment by stoty** [comment]
> Here we need manual verification only with locally deployed cluster?

Yes, that's the only way to make sure.

**Comment by stoty** [comment]
The most frustrating part is that every single one of these Guava usages is completely unneccessary, this could have been done without Guava with the same effort.
But I digress.

**Review by chrajeshbabu — APPROVED** [review]

**Comment by NihalJain** [comment]
I see there is an explicit guava dependency in omid at https://github.com/apache/phoenix-omid/blob/master/pom.xml#L1215. If it was replaced with shaded one, do we really need it?

**Comment by NihalJain** [comment]
I think this PR also handles thttps://issues.apache.org/jira/browse/OMID-266, just FYI @stoty

**Comment by stoty** [comment]
> I see there is an explicit guava dependency in omid at https://github.com/apache/phoenix-omid/blob/master/pom.xml#L1215. If it was replaced with shaded one, do we really need it?

That's not a dependency, just a dependency management entry.
It's not strictly necessary, but it prevents including old Guava versions in case some dependency decides to depend on unrelocated guava.

**Comment by stoty** [comment]
I have successfully tested this on a pseudistributed cluster with Phoenix HEAD.

**Comment by stoty** [comment]
This was merged a long time ago, closing the PR.

**Pull request closed without merging**